core/src/main/java/ai/timefold/solver/core/config/heuristic/selector/move/generic/list/AdvancedListRuinRecreateMoveSelectorConfig.java

```diff
@@ -0,0 +1,119 @@
+package ai.timefold.solver.core.config.heuristic.selector.move.generic.list;
+
+import java.util.function.Consumer;
+
+import jakarta.xml.bind.annotation.XmlType;
+
+import ai.timefold.solver.core.config.heuristic.selector.move.MoveSelectorConfig;
+import ai.timefold.solver.core.config.util.ConfigUtils;
+
+import org.jspecify.annotations.NonNull;
+
+@XmlType(propOrder = {
+        "ruinAndRecreatePickerClass",
```

Comment:
In the interest of brevity, why not just "pickerClass"?
We're already configuring RuinRecreateMoveSelector, I don't think we need to repeat that over and over.